protocol/avdl/chat1/local.avdl

```diff
@@ -22,7 +22,10 @@ protocol local {
   }
 
   record MessageAttachment {
-    string path;
+    string storageType;  // where the attachment object is stored (i.e. s3)
```

Comment:
Good point.  Could then remove the encryption version int as that will be implied by the version of the attachment struct.